index.html

```diff
@@ -1,158 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-	<head>
-		<meta charset="UTF-8" />
-		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
-		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<link
-			rel="stylesheet"
-			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
-			integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
-			crossorigin="anonymous"
-			referrerpolicy="no-referrer"
-		/>
-		<title>Fork Me! FCC: Test Suite Template</title>
-		<link rel="stylesheet" href="css/style.css" />
-	</head>
-
-	<body>
-		<!-- header -->
-		<header id="header">
-			<!-- logo -->
-			<div class="logo">
-				<img src="img/logo.png" id="header-img" alt="" />
-			</div>
```

Comment:
You should complete "alt" attribute which isn't mandatory but is incredibly useful for accessibility — screen readers read this description out to their users so they know what the image means.